data/human/hails.txt

```diff
@@ -1568,6 +1569,107 @@ phrase "friendly civilian"
 		"Apparently people think we'll work for exposure."
 		"Not to mention this ship is not designed for a job like it."
 
+phrase "friendly civilian"
+	word
+		"Don't go for"
+		"Don't participate in"
+		"Stay away from"
+		"Watch out for"
+	word
+		" "
+	word
+		"badly organized races"
+		"inheritance scams"
+		"shady transport jobs"
+		"hitman transports"
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"I lost 20000 credits on one of those."
+		"I nearly got killed in one."
+		"The Navy almost arrested me."
+		"Those can ruin your life!"
+		"They are usually too good to be true."
+		"They take your money and you won't get anything in return."
+
+phrase "friendly civilian"
+	word
+		"I have a delivery for"
+		"I'm looking for"
+		"My passengers need to go to"
+	word
+		" "
+	word
+		"Dune"
+		"Earth"
+		"Foundry"
+		"Glaze"
+		"Moonshake"
+		"Norn"
+		"Skillet"
+	word
+		", but my starmap doesn't list it."
+		"; do you know where I can find that planet?"
+		"; have you ever heard of a planet with that name?"
+		"; that planet shouldn't be too far from here."
+		"; which hyperlane should I take to get there?"
+
+phrase "friendly civilian"
+	word
+		"I almost collided with"
+		"I bumped into"
+		"I found"
+		"I picked up"
+	word
+		" "
+	word
+		"an antique teapot"
+		"a convertable car"
+		"some old memory capsule"
+	word
+		" "
+	word
+		"between Earth and Mars"
+		"around Earth"
+		"around Mars"
+	word
+		"."
+		"!"
+	word
+		""
+		"Can you believe that?"
+		"Do you want to see it?"
+		"It was in an unstable orbit."
+		"Must have been launched in the old days."
+		"Unbelieveable what people sent into space."
+
+phrase "friendly civilian"
+	word
+		"Check out the"
+		"Look for the"
+	word
+		" "
+	word
+		"race that is organized"
+		"stunt pilots flying in formation"
+	word
+		" "
+	word
+		"in the Deep"
+		"in Syndicate space"
+		"on Earth Day"
```

Comment:
Could this cause some continuity issues with `last month` below since this hail can appear at any time of year? Maybe you could replace it with something like I've suggested below:
```suggestion
		"through the paradise worlds"
```